packages/experience/package.json

```diff
@@ -56,6 +56,7 @@
     "camelcase-keys": "^9.0.0",
     "classnames": "^2.3.1",
     "color": "^4.2.3",
+    "core-js": "^3.34.0",
```

Comment:
would you mind running a `pnpm i`?  So the lock file may pick up the new dependency changes.

Reply:
This should fix some of the ci failures.

Reply:
It was fixed.